code/modules/mod/modules/modules_engineering.dm

```diff
@@ -85,7 +85,7 @@
 		these are only capable of working in zero-gravity environments, a blessing to some Engineers."
 	icon_state = "tether"
 	module_type = MODULE_ACTIVE
-	complexity = 3
+	complexity = 1
```

Comment:
keep this at 2

Reply:
Why? This is literally an inbuilt fire extinguisher dude. It doesn't even reach as far as one.

Reply:
what does this even mean, it's longer than your view range both horizontally and vertically

Reply:
Feels like this module will pretty much always be quite useless as long as we can still grab one credit out of the ID and use that as propulsion

Reply:
I'm not fumbling for credits if I could just use the tether to jettison myself along during zero grav. The complexity definitely is the real holding this module back, it has value once you start seeing some of the neat combinations it has with other modules.